.github/workflows/orchestrion.yml

```diff
@@ -0,0 +1,21 @@
+name: Orchestrion
+on:
+  workflow_dispatch: # manually
+  schedule: # nightly
+    - cron: "0 0 * * *"
```

Comment:
Is this really necessary? Do we somehow get notified when this fails? If not, nobody will be looking at it, so it's a waste of resources...